src/core/Makefile.am

```diff
@@ -81,6 +81,7 @@ libopenthread_a_SOURCES             = \
     thread/panid_query_server.cpp     \
     thread/thread_netif.cpp           \
     thread/thread_tlvs.cpp            \
+    utils/global_address.cpp          \
```

Comment:
What is the content of <UniqueIdentifier> tag in .filters file? How can I create it?